graylog2-web-interface/src/components/systemmessages/SystemMessagesList.jsx

```diff
@@ -1,31 +1,43 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import styled from 'styled-components';
+
 import { Table } from 'components/graylog';
 
 import { SystemMessage } from 'components/systemmessages';
 
-class SystemMessagesList extends React.Component {
-  static propTypes = {
-    messages: PropTypes.arrayOf(PropTypes.object).isRequired,
-  };
-
-  render() {
-    return (
-      <Table className="system-messages" striped hover condensed>
-        <thead>
-          <tr>
-            <th style={{ width: '200px' }}>Timestamp</th>
-            <th>Node</th>
-            <th>Message</th>
-          </tr>
-        </thead>
-
-        <tbody>
-          {this.props.messages.map(message => <SystemMessage key={`message-${Math.random().toString(36).substring(7)}`} message={message} />)}
-        </tbody>
-      </Table>
-    );
+const SystemMessagesTable = styled(Table)`
+  font-size: 12px;
+
+  a {
+    color: #000;
   }
-}
+`;
+
+const TimestampTH = styled.th`
+  width: 200px;
+`;
+
+const SystemMessagesList = ({ messages }) => {
+  return (
+    <SystemMessagesTable className="system-messages" striped hover condensed>
```

Comment:
I think we don't need the `className` here any more, and same goes with the style definition in the global styles.